data/software.yml

```diff
@@ -593,7 +593,7 @@ Middleware and Identity:
 
   - name: Keycloak
     image: keycloak
-    url: https://keycloak.jboss.org/
+    url: http://keycloak.org/
```

Comment:
The site has moved to Keycloak.org. HTTPS does not work, however, so I had to also change it to HTTP.